frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts

```diff
@@ -59,7 +60,8 @@ describe('Create VM from Template using wizard', () => {
             .setDisks(vmt.getData().disks)
             .build();
           await withResource(leakedResources, vm.asResource(), async () => {
-            await vm.create();
+            await vmt.createVMFromRowLink();
```

Comment:
create should be here. Please see https://github.com/openshift/console/pull/6850/files#r503824368